src/args.rs

```diff
@@ -1,21 +1,36 @@
 use super::Command;
 
+extern crate atty;
```

Comment:
I _think_ we can safely remove this line.

Reply:
Oh yep, good call!